src/components/maincontroller.ts

```diff
@@ -701,8 +716,13 @@ export function setTextTrack(index) {
     }
 }
 
-export function createMediaInformation(playSessionId, item, streamInfo) {
-    var mediaInfo = new cast.framework.messages.MediaInformation();
+// TODO no any types
+export function createMediaInformation(
+    playSessionId: string,
+    item: BaseItemDto,
+    streamInfo: any
```

Comment:
Same as above

Reply:
The whole streamInfo construct is quite random to me, I think they just put whatever in there so it's definitely a new type